index.html

```diff
@@ -85,242 +95,270 @@ <h2>LET'S CONNECT</h2>
-            <a href="#footer-icon">
-              <img src="./images/Linkedin icon.svg" alt="Linkedin logo"
-            />
-            </a>
-          </li>
-          <li>
-            <a href="#footer-icon"
-              ><img src="./images/hand-peace-icon.svg" alt="peace logo"
-            /></a>
-          </li>
-          <li>
-            <a href="#footer-icon"
-              ><img src="./images/twitter (2).svg" alt="twitter logo"
-            /></a>
-          </li>
-          <li>
-            <a href="#footer-icon"
-              ><img src="./images/m-icon.svg" alt="m logo"
-            /></a>
-          </li>
-        </ul>
-      </div>
-      <button  id="btn-get" class="see-pro .btn">Get my resume</button>
-      <div class="frame-lang">
-        <h2 class="language">Languages</h2>
-        <img src="./images/Icon-Right-Larger.svg" alt="" />
-      </div>
-      <div class="list-skills-about">
-        <ul>
-          <li>
-            <img class="img-js" src="./images/javascript.svg" alt="javascript" />
-          </li>
-          <li><img class="img-html" src="./images/html.svg" alt="html" /></li>
-          <li><img class="img-css" src="./images/css.svg" alt="css" /></li>
-        </ul>
-      </div>
-      <div class="frame">
-        <p>Frameworks</p>
-        <img src="./images/Icon-Right-Larger.svg" alt="" />
-      </div>
-      <div class="frame">
-        <p>Skills</p>
-        <img src="./images/Icon-Right-Larger.svg" alt="" />
-      </div>
-    </section>
-<!-- ===========CONTACT-ME======= -->
-<fieldset>
-  <section class="contact-me">
-    <div class="text-field hello">
-      <h2>Contact me</h2>
-      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-    </div>
-    <div class="input-field">
-      <form action="https://formspree.io/f/mnqynlww" method="post">
-        <input class="input-design" type="text" name="full_name" id="full_name" placeholder="Enter your Name" maxlength="30" required>
-        <input class="input-design" type="email" name="email" id="email" placeholder="Email Address" required>
-        <textarea class="textArea" name="message"  cols="30" rows="10" minlength="" maxlength="500" placeholder="write your message here" required></textarea>
-        <input  type="submit" value="Get in touch" class="see-pro submit-btn">
-      </form>
-      <div class="bottom_line"></div>
+      <div class="right-desktop">
+        <div class="about">
+          <h1>
+            About <br />
+            Myself
+          </h1>
+        </div>
+        <div class="hello hide-overflow">
+          <p >
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </div>
```

Comment:
- [ ] You should ensure that you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). I, therefore, suggest reducing the size of the title and paragraph slightly 👍 🥰 . This will make the UI user-friendly and professional ☺️ .
- [ ] I have noticed that the `line-height` given to the `.hello p` is making the text-overflow. Kindly ensure that the text can be able to stretch when reducing the screen size. This will make the website more accessible 🥰 .
  - Your design 😎 
![abt](https://user-images.githubusercontent.com/99394374/215081736-058b2334-a35f-4ed8-aa48-280a25b46fe0.PNG)


   - Figma design 😎 
![abt lang](https://user-images.githubusercontent.com/99394374/215081787-714cea3a-bbfc-44f7-aa65-636cc950b326.PNG)